### PR TITLE
feat: Admin workloads Namespace

### PR DESCRIPTION
**Pull request by leiarenee**
Following features were implemented:

- Create applications which can have cluster wide resources
- Create applications in namespaces other than project ns

## Checklist

- [x] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes.

### CODE REVIEW

File: charts/team-ns/templates/argocd/argocd-applicationset.yaml [hunk 1]
```diff
@@ -45,7 +45,13 @@ spec:
         syncOptions: []
       destination:
         server: 'https://kubernetes.default.svc'
-        namespace: team-{{ $v.teamId }} # namespace is restricted
+        # namespace is restricted to team-<teamId> for team members. For team admins '$v.admin.namespace' is used.
```
Comment (j-zimnowoda) [inline charts/team-ns/templates/argocd/argocd-applicationset.yaml:48]: this comment is unnecessary as the JSCON schema description explains that

**Comment by j-zimnowoda** [comment]
Could you add test/fixtures for admin workloads with and without namespace ?

**Comment by leiarenee** [comment]
> Could you add test/fixtures for admin workloads with and without namespace ?

It's done

**Comment by j-zimnowoda** [comment]
perfect!

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**